helm/operator/templates/console-deployment.yaml

```diff
@@ -58,7 +58,15 @@ spec:
                 {{ toYaml . | nindent 12 }}
                 {{- end }}
           resources:
-      {{- toYaml .Values.console.resources | nindent 12 }}
+          {{- toYaml .Values.console.resources | nindent 12 }}
+          volumeMounts:
+          {{- with .Values.console.volumeMounts }}
+          {{- toYaml . | nindent 12 }}
+          {{- end }}
+      volumes:
+      {{- with .Values.console.volumes }}
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
       {{- with .Values.console.initContainers }}
```

Comment:
what kind of secrets are being mounted here? - there is nothing special about Operator console that needs to mount secrets here @bradthebuilder

Reply:
I needed to mount a TLS Secret to integrate MinIO Console with Keycloak's SSO

Reply:
you mean operator console @bradthebuilder ? -  @Alevsk do we support external SSO with Operator UI? - if yes can you check here if this is allowed ?

Reply:
Yes indeed, the Operator Console. "Support" and "technically possible" may be distinct, but I can attest to the latter with this PR's changes. I use the CONSOLE_IDP_* environment variables mentioned in issue #603 and mount in my CA chain to prevent "unrecognized certificate" errors when the Console attempts to redirect to Keycloak.
```
env:
- name: CONSOLE_IDP_URL
  value: ...
- name: CONSOLE_IDP_CLIENT_ID
  value: ...
- name: CONSOLE_IDP_SECRET
  value: ...
- name: CONSOLE_IDP_CALLBACK
  value: ...
volumes:
  - name: ca-cert
    secret:
    secretName: my-ca-chain
    defaultMode: 0644
volumeMounts:
  - name: ca-cert
    mountPath: /etc/ssl/certs
    readOnly: true
```
Pictoral evidence of the configuration replacing the JWT login prompt with an SSO prompt:
![minio-operator-console-sso](https://user-images.githubusercontent.com/49787611/149340417-3076a7a3-9436-4f6d-915d-9802f71a94ae.png)